packages/wonder-blocks-breadcrumb/components/breadcrumb.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import type {AriaProps} from "@khanacademy/wonder-blocks-core";
+import {addStyle} from "@khanacademy/wonder-blocks-core";
+import BreadcrumbItem from "./breadcrumb-item.js";
+
+type Props = {|
+    ...AriaProps,
+
+    /**
+     * This is the content for the collection of Breadcrumbs
+     */
+    children: React.ChildrenArray<React.Element<typeof BreadcrumbItem>>,
+|};
+
+const StyledList = addStyle("ol");
+
+/**
+ * The Breadcrumb component will have the following structure:
+ *
+ * 1. Breadcrumb Item: Represents a section within the page.
+ * 2. Separator: Adds a separator between each item.
+ */
+export default class Breadcrumb extends React.Component<Props> {
```

Comment:
I think we may want to change the name of the component to `Breadcrumbs` and the package to `wonder-blocks-breadcrumbs` since people usually use the plural form.

Reply:
@kevinbarabash for a single item, do you think it should be `BreadcrumbsItem` or `BreadcrumbItem?

Reply:
I think probably `BreadcrumbItem`.